app/controllers/posts_controller.rb

```diff
@@ -4,11 +4,11 @@ class PostsController < ApplicationController
 
   def index
     @posts = Post.all
-    render json: @posts
+    render 'index'
   end
 
   def show
-    render json: @post
+    render 'show'
```

Comment:
[OPTIONAL]
- [ ] Same here as pointed in the previous comment :point_up_2: rails will render the `app/views/posts/show.html.erb` by default for the `show` action in the posts controller. Therefore I suggest you remove this `render 'show'`

Reply:
👍